lib/Nexus.js

```diff
@@ -268,6 +269,42 @@ module.exports = function xGraph(__options = {}) {
 				return pid;
 			}
 
+			/**
+			 * Scan the array of Pids and send a 'GetServices'
+			 * command to each in turn to accumulate the 
+			 * services used by a partifcular module
+			 * @param {array} services		array of Pids to send GetServices to
```

Comment:
services is not a descriptive variable name here. its an array of pids, maybe `pidArr`, or `pids`